debian/kubelet/kubelet-wrapper

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+source /etc/default/kubelet
```

Comment:
By the way, it looks like this file seems to be missing from the package, and this script will fail, unless user has created it.